assets/css/project.css

```diff
@@ -402,7 +402,7 @@ input[type="text"]{
 
 @media only screen and (max-width: 702px) {
   .NeuInput-outer {
-    width: 300px;
+    /* width: 300px; */
```

Comment:
Remove unnecessary lines instead of commenting them. Later on it will be easier

Reply:
okay